pkg/distro/rhel9/edge.go

```diff
@@ -586,7 +586,7 @@ func edgeCommitPackageSet(t *imageType) rpmmd.PackageSet {
 		ps = ps.Append(aarch64EdgeCommitPackageSet(t))
 	}
 
-	if !common.VersionLessThan(t.arch.distro.osVersion, "9.2") || !common.VersionLessThan(t.arch.distro.osVersion, "9-stream") {
+	if !common.VersionLessThan(t.arch.distro.osVersion, "9.2") || t.arch.distro.osVersion == "9-stream" {
```

Comment:
I think I prefer `!t.arch.isRHEL()` for determining if a distro is Centos Stream, but it's also a bit weird. Maybe we should introduce a `isCentosStream` helper at some point.

Reply:
I fixed all comparisons of the `osVersion` to `9-stream`.